cmds/help.js

```diff
@@ -91,20 +91,22 @@ function cmd_help(lang, msg, args, line, wiki) {
 		var cmd = ( lang.aliases[invoke] || invoke );
 		if ( cmd === 'admin' ) {
 			if ( !msg.channel.isGuild() || msg.isAdmin() ) {
-				var cmdlist = lang.get('help.admin') + '\n';
-				cmdlist += formathelp(helplist.admin, msg, lang);
+				var embed = new MessageEmbed().setTitle( lang.get('help.admin') ).setFooter( 'Admin Help' ).setTimestamp();
+				var cmdlist = formathelp(helplist.admin, msg, lang);
 				cmdlist += '\n\n🔸 ' + lang.get('help.adminfooter');
 				cmdlist += '\n\t\t' + new URL(( msg.channel.isGuild() ? `/guild/${msg.guild.id}/settings` : '/' ), process.env.dashboard).href;
-				msg.sendChannel( cmdlist, {split:{char:'\n🔹',prepend:'🔹',maxLength}} );
+				embed.setDescription( cmdlist );
+				msg.replyMsg( '', {embed}, false, false )
```

Comment:
This will break for languages where the `cmdlist` is above the maximal character limit as it is no longer split into multiple messages.